app/templates/components/article-comment.hbs

```diff
@@ -1,5 +1,5 @@
 <div class='avatar'>
```

Comment:
Can it check if only double quotes are used for attribute values?

Reply:
it will be able in next version